tools/bifrost-benchpress/src/append_latency.rs

```diff
@@ -10,46 +10,65 @@
 
 use std::time::Instant;
 
+use bytes::BytesMut;
 use hdrhistogram::Histogram;
 use tracing::info;
 
 use restate_bifrost::Bifrost;
 use restate_core::TaskCenter;
-use restate_types::logs::LogId;
+use restate_types::logs::{LogId, WithKeys};
 
-use crate::util::print_latencies;
+use crate::util::{print_latencies, DummyPayload};
 use crate::Arguments;
 
+const LOG_ID: LogId = LogId::new(0);
```

Comment:
not that it would matter much: `const { ... }`?

Reply:
That'd only matter for `static` not for consts.